org.musescore.MuseScore.yml

```diff
@@ -44,49 +45,33 @@ cleanup-commands:
 modules:
   - shared-modules/lame/lame-3.99.5.json
 
-  - name: portaudio
-    config-opts:
-      - --disable-static
-      - --without-oss
-    sources:
-      - type: archive
-        url: http://www.portaudio.com/archives/pa_stable_v190600_20161030.tgz
-        sha256: f5a21d7dcd6ee84397446fa1fa1a0675bb2e8a4a6dceb4305a8404698d8d1513
+  - name: jack2
```

Comment:
1. PortMIDI isn’t irrelevant, it enables real-time MIDI input _without_ JACK.

2. I don’t think JACK actually works in Flatpak, it certainly didn’t last time I checked.